src/components/Calculator.js

```diff
@@ -1,69 +1,67 @@
 /* eslint-disable linebreak-style */
 /* eslint-disable indent */
 /* eslint-disable react/prefer-stateless-function */
```

Comment:
- [ ] Disabling linters defeat the ultimate purpose of linters, instead I think you should try removing these 3 lines and fixing any problems that may arise.

Reply:
- [ ] Let's change your App component back to a stateless functional component as this is a best practice nowadays if the components do not have any state.

Reply:
[OPTIONAL] I think there are many files from the boilerplate that `create-react-app` creates. You could try to remove as many as possible to clean up the repository so that this remains professional.

Reply:
Thank you @D4nielJ